src/main/webapp/index.html

```diff
@@ -31,7 +31,7 @@
     <h1>CodeU Starter Project</h1>
     <p>This is the CodeU starter project. Click the links above to login and visit your page.
     You can post messages on your page, and you can visit other user pages if you have
-    their URL.</p>
+    their URL. Hello World!</p>
```

Comment:
Can you remove this "Hello World" string.

Reply:
this looks good, other than the thing above. I compared to the one i have and the one on the website. ! good job !